pyweek36/constants.py

```diff
@@ -67,7 +73,13 @@ class InputType(IntEnum):
 
 ASSETS_DIR = Path(__file__).parent.parent / "assets"
 DARKMATTER_TEXTURE_PATH = ASSETS_DIR / "sprites/map/dark_matter.png"
+WALL_TEXTURE_PATH = ASSETS_DIR / "sprites/map/solid_block.png"
 PLAYER_IDLE_ANIM_PATH = ASSETS_DIR / "player/idle"
 PLAYER_JUMP_ANIM_PATH = ASSETS_DIR / "player/jump"
 PLAYER_WALK_ANIM_PATH = ASSETS_DIR / "player/walk"
 LOOPING_TEXTURES = {"idle", "walk"}
+
+# Load texture now so it isn't loaded from disk every frame
+DARKMATTER_TEXTURE = load_texture(DARKMATTER_TEXTURE_PATH)
+
+WALL_TEXTURE = load_texture(WALL_TEXTURE_PATH)
```

Comment:
I feel like loading textures shouldn't be done in `constants.py`, I think loading it when creating the class as a class attribute (like in `PlayerSprite`) would be better

Reply:
Or if we're going to load assets for tiles here, we may as well load all the assets here